cmd/geth/consolecmd_test.go

```diff
@@ -79,7 +79,7 @@ func runMinimalGeth(t *testing.T, args ...string) *testgeth {
 	// --ropsten to make the 'writing genesis to disk' faster (no accounts)
 	// --networkid=1337 to avoid cache bump
 	// --syncmode=full to avoid allocating fast sync bloom
-	allArgs := []string{"--ropsten", "--nousb", "--networkid", "1337", "--syncmode=full", "--port", "0",
+	allArgs := []string{ /*"--ropsten",*/ "--nousb", "--networkid", "1337", "--syncmode=full", "--port", "0",
```

Comment:
--ropsten is failing the test, we may remove the comment

Reply:
add an extra comment to say that is a Quorum modification

Reply:
also, which tests fails?

Reply:
FAIL: TestConsoleWelcome (0.13s)
`Failed to register the Ethereum service: database contains incompatible genesis (have 05770abd0d86be84e155ce0c9f32024b5884bc6ae025b44619baf11708f2b05f, new 41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d)`